components/03-modules/additional-content/_additional-content.scss

```diff
@@ -0,0 +1,24 @@
+$additional-content__image-margin          : 0 0 41px !default;
+$additional-content__image-margin--screen-l: 0 0 32px !default;
```

Comment:
Should be with `@`. Its new code, therefore, you can change it.

Reply:
sure